internal/storage/driver/postsql/operation.go

```diff
@@ -169,6 +169,61 @@ func (s *operations) ListOperationsByInstanceID(instanceID string) ([]internal.O
 	return ret, nil
 }
 
+func (s *operations) ListOperationsByInstanceIDGroupByType(instanceID string) (*internal.GroupedOperations, error) {
```

Comment:
Is this really faster when we fetch all and group and sort in memory?

Reply:
So if we fetch operations for particular instanceID this approach make sense.
But where do we need such data?
Operations per instanceID grouped by type?

Reply:
the response of runtimes endpoint groups operations by type